cluster/addons/node-problem-detector/npd.yaml

```diff
@@ -26,24 +26,22 @@ subjects:
 apiVersion: apps/v1
 kind: DaemonSet
 metadata:
-  name: npd-v0.8.9
+  name: node-problem-detector
   namespace: kube-system
   labels:
-    k8s-app: node-problem-detector
-    version: v0.8.9
-    kubernetes.io/cluster-service: "true"
+    app.kubernetes.io/name: node-problem-detector
+    app.kubernetes.io/version: v0.8.13
     addonmanager.kubernetes.io/mode: Reconcile
 spec:
   selector:
     matchLabels:
-      k8s-app: node-problem-detector
-      version: v0.8.9
```

Comment:
may be a good idea to fix up https://github.com/kubernetes/kubernetes/blob/e314ec2c9895902c7b23b65fcfb1d64a3ee0701a/build/dependencies.yaml#L91 to include these versions into the regex list

Reply:
`build/dependencies.yaml` is using to fail the CI if places in regex list are not updated with other parts.
- adding all version related line to the regex list can avoid inconsistent when we upgrade npd the next time.

Reply:
The expected change is like https://github.com/kubernetes/kubernetes/compare/master...pacoxu:kubernetes:patch-7

Reply:
Done

Reply:
Done